server/models/establishment.js

```diff
@@ -1618,5 +1618,53 @@ module.exports = function (sequelize, DataTypes) {
     });
   };
 
+  Establishment.getWorkersWithCareCertificateStatus = async function (establishmentId, isParent = false) {
+    let childEstablishments = [];
+
+    if (isParent) {
+      childEstablishments = [
+        {
+          parentId: establishmentId,
+          dataOwner: 'Parent',
+        },
+        {
+          parentId: establishmentId,
+          dataOwner: 'Workplace',
+          dataPermissions: 'Workplace and Staff',
+        },
+      ];
+    }
+
+    return this.findAll({
+      attributes: ['id', 'NameValue'],
+      where: {
+        [Op.or]: [
+          {
+            id: establishmentId,
+          },
+          ...childEstablishments,
+        ],
+      },
+      include: [
+        {
+          model: sequelize.models.worker,
+          as: 'workers',
+          attributes: ['NameOrIdValue', 'CareCertificateValue'],
```

Comment:
You will probably need to add `required: false` here too

```suggestion
          attributes: ['NameOrIdValue', 'CareCertificateValue'],
          required: false
```